server/controllers/PokerController.js

```diff
@@ -0,0 +1,216 @@
+const { v1: uuid } = require('uuid');
+let PokerUtils = require('../utilities/PokerUtils.js');
+let PokerTable = require("../model/PokerTable.js");
+const DataAccessLayer = require('../controllers/DataAccessLayer.js');
+
+class PokerController {
+    constructor(numberOfEachRoom) {
+        this.pokerTableStats = PokerUtils.GetPokerTableStats();
+        this.numberOfEachRoom = numberOfEachRoom;
+        this.rooms = [];
+
+        for (let i = 0; i < this.pokerTableStats.length; i++) { // for each type of table we create a few
+            for(let j = 0; j < this.numberOfEachRoom; j++) { // right now two of each table is created
+                let room = {
+                    id: uuid(),
+                    table: new PokerTable(this.pokerTableStats[i]),
+                }
+                this.rooms[i*this.numberOfEachRoom + j] = room;
+            }
+        }
+    }
+
+    /**
+     * Checks if a User can join the room, ********needs to use the real login and tables page
```

Comment:
What's with all the asterisks? Should this be a TODO tag?

Reply:
Fixed, added a TODO tag